### DISCUSSION TITLE
Flexsearch doesn't work in latest version, what I am missing?

### DISCUSSION THREAD

**Discussion by mursu83** [thuliteio/doks#416] [Q&A]
I'm not sure where I should place this problem, but I recently updated Doks to its latest version. For some reason, Flexsearch doesn't function properly anymore. It has been enabled properly in config.toml and when site is published, search index is generated correctly to index.js. However, if I try type something into search bar, it doesn't show anything, as if CSS for showing the results would be missing. 

What makes the situation even more bizarre is that I did systematically compare related source files in both old and new Doks versions. Outside main.css, there are no differences in used files / generated outputs. Nevertheless, flexsearch functions just fine in the older version. Any idea about what I am missing? I am generating my site on Windows 8.1.

**Comment by h-enk** [comment]
Please share a link to your repository

**Reply by mursu83** [reply]
Hi,

I don't have repository, but I found solution to the problem. The issue seems to actually have nothing to do with Doks, but Windows version of Hugo. For some reason, when it generates fingerprints for publication, it compiles files incorrectly, adding random line breaks and white spaces in places where they shouldn't exist. Removing fingerprint generation from CSS and JS scripts solved the problem.  

Thank you for your assistance.
